map_model/src/traversable.rs

```diff
@@ -70,29 +70,32 @@ impl Position {
         }
     }
 
+    /// Given a position along a lane, find the equivalent position along another lane on the same
```

Comment:
No behavioral change here. While refactoring for `Direction`, I just made the variable names less confusing.